.github/workflows/ci.yml

```diff
@@ -35,13 +35,13 @@ jobs:
         run: swift package generate-xcodeproj --output generated/ && carthage build --project-directory generated/ --verbose --no-skip-current
   spm-11:
     name: SPM Build Xcode 11
-    runs-on: macOS-10.15
+    runs-on: macOS-11
     strategy:
       matrix:
         platforms: [
           'iOS_12,tvOS_12',
           'iOS_13,tvOS_13,watchOS_6',
-          'macOS_10_15'
+          'macOS_11'
```

Comment:
I don't love this change but not much we can do about it. We still technically support back to macOS 10.14, even though we haven't been able to run CI on that OS in years 😬